docs/manual/src/tutorial/udl_file.md

```diff
@@ -1,10 +1,19 @@
 # Describing the interface
 
 There are two ways of describing your interface:
-1 - with a UDL file (a type of IDL, Interface Definition Language);
-2 - with [Rust procmacros](../proc_macro/index.md), similarly to how `wasm-bindgen` work. This avoids repeating your interface definitions in a separate file. Unfortunately, our docs aren't quite as good for that yet though.
 
-## The UDL File
+## Procmacros
+
+[Rust procmacros](../proc_macro/index.md) can describe your interface - you'd define the function as:
```

Comment:
Nit-pick, but we use `proc macro` in other places. 

```suggestion
## Proc macros

[Rust proc macros](../proc_macro/index.md) can describe your interface - you'd define the function as:
```

But I guess either is fine, so feel free to ignore.